openpype/modules/kitsu/plugins/publish/integrate_kitsu_review.py

```diff
@@ -12,17 +12,17 @@ class IntegrateKitsuReview(pyblish.api.InstancePlugin):
     optional = True
 
     def process(self, instance):
-        task = instance.data["kitsu_task"]["id"]
-        comment = instance.data["kitsu_comment"]["id"]
 
         # Check comment has been created
+        comment = instance.data.get("kitsu_comment", {}).get("id")
         if not comment:
             self.log.debug(
                 "Comment not created, review not pushed to preview."
             )
             return
 
         # Add review representations as preview of comment
+        task = instance.data["kitsu_task"]["id"]
```

Comment:
As we are dealing with IDs here, and refactoring for better understanding, I'd rename them for `comment_id` and `task_id`.

Reply:
+1 for this

Reply:
Done.